troubleshooting/known_issues/orphaned_cfs_pods.md

```diff
@@ -0,0 +1,34 @@
+# Orphaned CFS Pods After Booting or Rebooting
+
```

Comment:
Instances of "Pod" should be "pod" to be consistent with the rest of the documentation.